yt/frontends/ramses/field_handlers.py

```diff
@@ -66,15 +66,29 @@ def __init__(self, domain):
         self.domain_id = domain.domain_id
         ds = domain.ds
         basename = os.path.abspath(
-              os.path.dirname(ds.parameter_filename))
+              ds.root_folder)
         iout = int(
             os.path.basename(ds.parameter_filename)
             .split(".")[0].
             split("_")[1])
 
-        self.fname = os.path.join(
-            basename,
-            self.fname.format(iout=iout, icpu=domain.domain_id))
+        if ds.num_groups > 0:
+            igroup = ((domain.domain_id-1) // ds.group_size) + 1
+            full_path = os.path.join(
+                basename,
+                'group_{:05d}'.format(igroup),
+                self.fname.format(iout=iout, icpu=domain.domain_id))
+        else:
+            full_path = os.path.join(
+                basename,
+                self.fname.format(iout=iout, icpu=domain.domain_id))
+
+        if os.path.exists(full_path):
+            self.fname = full_path
+        else:
+            raise FileNotFoundError(
```

Comment:
Looks like there's a py2 issue:

```
AssertionError: flake8 found style errors:
c:\projects\yt-1nc0i\yt\frontends\ramses\field_handlers.py:89:19: F821 undefined name 'FileNotFoundError'
```

you have some code below to handle this in another file, I think it just needs to be added to this file.